distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -66,30 +48,128 @@ a|* Does not scale. Only runs one single server instance.
+
+.Solr Catalog External Provider Pros and Cons
+[cols="1,4a,2a" options="header"]
+|===
+
+|Feature
+|Pro
+|Con
+
+|Scalability
+a|* Allows the middle-tier to be scaled by pointing various middle-tier instances to one server facade.
+* Possible data tier scalability with Solr Cloud. Solr Cloud allows for "high scale, fault tolerant, distributed indexing and search capabilities.
+a|
+
+|Flexibility
+a|* REST-like HTTP/XML and JSON APIs that make it easy to use from virtually any programming language.
+* Ability to run in separate or same JVM of middle tier.
+a|
+
+|(Administrative)Tools
+a|* Contains Solr Admin GUI, which allows admins to query, check health, see metrics, see configuration files and preferences, etc.
+* JMX metrics reporting is enabled.
+a|
+
+|Security
+a|* Inherits app server security.
+a|* Web Console must be secured and is openly accessible.
+* REST-like HTTP/XML and JSON APIs must be secured.
+* Current Catalog Provider implementation requires sending unsecured messages to Solr. Without a coded solution, requires network or firewall restrictions in order to secure.
  
 |Performance
-a|* Requires no HTTP or network overhead
-* Near real-time indexing
+a|* Near real-time indexing.
+a|* Possible network latency impact.
+* Extra overhead when sent over HTTP.
+* Extra parsing for XML, JSON, or other interface formats.
+* Possible limitations upon requests and queries dependent on HTTP server settings.
+
+|Backup/Recovery
+a|* Built-in recovery tools that allow in-place backups (does not require server shutdown).
+* Backup of Solr indexes can be scripted.
+a|* Recovery is performed as an HTTP request.
+
+|===
+
+==== SolrCloud
+
+If the ${branding} needs to be accessible for a higher than normal period with low instances of downtime, then SolrCloud should be used.
+
+SolrCloud is a cluster of Solr Server instances that support fault tolerance and high availability.
+Each Solr Server instance in SolrCloud provides a Solr Admin GUI.
+
+Synchronization between Solr Server instances is managed by Zookeeper.
+Zookeeper routes client requests to the proper Solr Server instance in the the SolrCloud cluster.
+
+.SolrCloud Pros and Cons
+[cols="1,4a,2a" options="header"]
+|===
+
+|Feature
+|Pro
+|Con
+
+|Scalability
+a|* Scalable
+* Can run a cluster of Solr Servers
+a|
+
+|Flexibility
+a|* Can be embedded in Java easily.
+* Uses the same interface as the Standalone Solr Server uses under the covers.
+|* Requires HTTP connection.
+* Setup and Installation is complicateed: needs Zookeeper to coordinate configurations between Solr Servers.
+* Synchronous issues on startup; i.e., Zookeeper nodes needs to be up and before SolrCloud is up.
+
+|(Administrative)Tools
+a|* Contains Solr Admin GUI, which allows admins to query, check health, see metrics, see configuration files and preferences, etc.
+a|
+
+|Security
+a|
+a|* SolrCloud is not secure, refer to SolrCloud Hardening Guide
```

Comment:
if "SolrCloud Hardening Guide" is a section header, you can just add: `<<_solrcloud_hardening_guide,SolrCloud Hardening Guide>>`, but if it's something external, just use the url, followed by the text you want in brackets: `http://example.com[Example Link]`